Makefile

```diff
@@ -308,554 +303,179 @@ test-integration: test-integration-gcp test-integration-humio test-integration-i
-stop-test-integration: stop-integration-loki stop-integration-mongodb_metrics stop-integration-nats
-stop-test-integration: stop-integration-nginx stop-integration-prometheus stop-integration-pulsar stop-integration-splunk
-
-.PHONY: start-integration-aws
-start-integration-aws:
-ifeq ($(CONTAINER_TOOL),podman)
-	$(CONTAINER_TOOL) $(CONTAINER_ENCLOSURE) create --replace --name vector-test-integration-aws -p 4566:4566 -p 4571:4571 -p 6000:6000 -p 9088:80
-	$(CONTAINER_TOOL) run -d --$(CONTAINER_ENCLOSURE)=vector-test-integration-aws --name vector_ec2_metadata \
-	 timberiodev/mock-ec2-metadata:latest
-	$(CONTAINER_TOOL) run -d --$(CONTAINER_ENCLOSURE)=vector-test-integration-aws --name vector_localstack_aws \
-	 -e SERVICES=kinesis,s3,cloudwatch,elasticsearch,es,firehose,sqs \
-	 localstack/localstack-full:0.11.6
-	$(CONTAINER_TOOL) run -d --$(CONTAINER_ENCLOSURE)=vector-test-integration-aws --name vector_mockwatchlogs \
-	 -e RUST_LOG=trace luciofranco/mockwatchlogs:latest
-	$(CONTAINER_TOOL) run -d --$(CONTAINER_ENCLOSURE)=vector-test-integration-aws -v /var/run:/var/run --name vector_local_ecs \
-	 -e RUST_LOG=trace amazon/amazon-ecs-local-container-endpoints:latest
-else
-	$(CONTAINER_TOOL) $(CONTAINER_ENCLOSURE) create vector-test-integration-aws
-	$(CONTAINER_TOOL) run -d --$(CONTAINER_ENCLOSURE)=vector-test-integration-aws -p 8111:8111 --name vector_ec2_metadata \
-	 timberiodev/mock-ec2-metadata:latest
-	$(CONTAINER_TOOL) run -d --$(CONTAINER_ENCLOSURE)=vector-test-integration-aws --name vector_localstack_aws \
-	 -p 4566:4566 -p 4571:4571 \
-	 -e SERVICES=kinesis,s3,cloudwatch,elasticsearch,es,firehose,sqs \
-	 localstack/localstack-full:0.11.6
-	$(CONTAINER_TOOL) run -d --$(CONTAINER_ENCLOSURE)=vector-test-integration-aws -p 6000:6000 --name vector_mockwatchlogs \
-	 -e RUST_LOG=trace luciofranco/mockwatchlogs:latest
-	$(CONTAINER_TOOL) run -d --$(CONTAINER_ENCLOSURE)=vector-test-integration-aws -v /var/run:/var/run -p 9088:80 --name vector_local_ecs \
-	 -e RUST_LOG=trace amazon/amazon-ecs-local-container-endpoints:latest
-endif
-
-.PHONY: stop-integration-aws
-stop-integration-aws:
-	$(CONTAINER_TOOL) rm --force vector_ec2_metadata vector_mockwatchlogs vector_localstack_aws vector_local_ecs 2>/dev/null; true
-ifeq ($(CONTAINER_TOOL),podman)
-	$(CONTAINER_TOOL) $(CONTAINER_ENCLOSURE) stop vector-test-integration-aws 2>/dev/null; true
-	$(CONTAINER_TOOL) $(CONTAINER_ENCLOSURE) rm --force vector-test-integration-aws 2>/dev/null; true
-else
-	$(CONTAINER_TOOL) $(CONTAINER_ENCLOSURE) rm vector-test-integration-aws 2>/dev/null; true
-endif
-
 .PHONY: test-integration-aws
 test-integration-aws: ## Runs AWS integration tests
 ifeq ($(AUTOSPAWN), true)
-	-$(MAKE) -k stop-integration-aws
-	$(MAKE) start-integration-aws
+	@scripts/setup_integration/setup_integration_env.sh -i aws -a stop -t ${CONTAINER_TOOL}
+	@scripts/setup_integration/setup_integration_env.sh -i aws -a start -t ${CONTAINER_TOOL}
```

Comment:
Since this `Makefile` exports `$CONTAINER_TOOL`, and `setup_integration_env.sh` appears to be able to use it without being specified as an option, I think the explicit `-t ${CONTAINER_TOOL}` option here is redundant.